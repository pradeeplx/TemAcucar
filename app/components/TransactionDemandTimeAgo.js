import React from 'react'
import { View } from 'react-native'
import Colors from "../Colors"
import Icon from "./Icon"
import TimeAgo from "./TimeAgo"

export default TransactionDemandTimeAgo = ({ demand: { created_at } }) => (
  <View style={{
    flexDirection: 'column',
    flex: 1,
    alignItems: 'flex-end',
    marginTop: 10,
  }}>
    <TimeAgo time={created_at} style={{color: Colors.black}} />
  </View>
)
