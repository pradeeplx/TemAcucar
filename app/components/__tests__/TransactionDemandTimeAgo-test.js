import 'react-native'
import React from 'react'
import TransactionDemandTimeAgo from '../TransactionDemandTimeAgo'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <TransactionDemandTimeAgo demand={{ created_at: 10E4 }} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
