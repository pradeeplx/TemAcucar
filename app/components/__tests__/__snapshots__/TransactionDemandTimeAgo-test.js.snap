exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-end",
      "flex": 1,
      "flexDirection": "column",
      "marginTop": 10,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Avenir",
          "fontSize": 18,
        },
        Array [
          Object {
            "fontSize": 13.5,
          },
          Object {
            "color": "#000000",
          },
        ],
      ]
    }>
    HÃ¡ 47 anos
  </Text>
</View>
`;
